lib/components/window/window_toolbar.dart

```diff
@@ -45,26 +45,25 @@ class _PangolinWindowToolbarState extends State<PangolinWindowToolbar> {
   late DragUpdateDetails _lastDetails;
   WindowDock? _draggingDock;
 
-
-
   @override
   Widget build(BuildContext context) {
     final hierarchy = WindowHierarchy.of(context);
     final properties = WindowPropertyRegistry.of(context);
     final layout = LayoutState.of(context);
     final fgColor = !context.theme.darkMode ? Colors.grey[900]! : Colors.white;
     final _customizationProvider = CustomizationProvider.of(context);
-Color widgetColor() {
-                if(_customizationProvider.coloredTitlebars){
-                  if(_customizationProvider.transparentColoredTitlebars && widget.barColor.opacity >= 0.5){
-                    return widget.barColor.op(0.5);
-                  }
-                  else{
-                    return widget.barColor;
-                  }
-                }
-                return Colors.transparent;
-}
+    Color widgetColor() {
+      if (_customizationProvider.coloredTitlebars) {
+        if (_customizationProvider.transparentColoredTitlebars &&
+            widget.barColor.opacity >= 0.5) {
+          return widget.barColor.op(0.5);
+        } else {
+          return widget.barColor;
+        }
+      }
+      return Colors.transparent;
+    }
```

Comment:
Not introduced in this pr, just noting: it should be avoided to have inner functions that return a widget like this, instead pass the elements they need from parameters